web/views/page_view.ex

```diff
@@ -4,4 +4,54 @@ defmodule Askbywho.PageView do
   """
 
   use Askbywho.Web, :view
+
+  alias Askbywho.Repo
+  alias Askbywho.Email
+  alias Askbywho.Brand
+
+  def count_emails do
+    Repo.aggregate Email, :count, :id
+  end
+
+  def count_brands_message([_]) do
+    "1 brand"
+  end
+
+  def count_brands_message(list) do
+    count = length list
+    "#{count} brands"
+  end
+
+  def count_brands do
+    Repo.aggregate Brand, :count, :id
+  end
+
+  def count_nominations do
+    {_status, result} = Ecto.Adapters.SQL.query(
+      Repo, "select count(1) from brands_emails"
```

Comment:
We typically shouldn't be putting Repo logic in the "views".  Typically the controller should load this information, and all calculation logic should be put in the Models themselves.  So, things like counts and percentage should all be gotten from the Repo, and then loaded as vars in the controller itself